vault/import_policy_test.go

```diff
@@ -0,0 +1,27 @@
+package vault
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccPolicy_importBasic(t *testing.T) {
+	name := "test-" + acctest.RandString(10)
```

Comment:
There's also [RandomWithPrefix](https://github.com/hashicorp/terraform/blob/master/helper/acctest/random.go#L30), if integers are valid in the name.

Reply:
TIL! Thanks :)